functions/routes/services/AwardBadgeService.js

```diff
@@ -31,21 +120,12 @@ const AwardService = {
   },
   listAwards: async (data, authToken) => {
     let response;
-
     await axios({
       headers: {
         Authorization: `Bearer ${authToken}`
       },
       method: 'get',
       url: `/issuers/${ISSUER_ID}/assertions`,
-      data: {
-        recipient: {
-          identity: "luis.alvarez@herounit.io",
-          type: 'email',
-          hashed: true
-        },
-        notify: true
-      }
     })
       .then(res => {
```

Comment:
Let's try to avoid using the promise callbacks. Prefer the async and await syntax.